docusaurus/docs/e2e-test-a-plugin/get-started.md

```diff
@@ -45,15 +50,9 @@ This article will guide through how to install and configure `@grafana/plugin-e2
 
 ### Installing Playwright
 
-Please refer to the [Playwright documentation](https://playwright.dev/docs/intro#installing-playwright) for instruction on how to install. `@grafana/plugin-e2e` extends Playwright APIs, so you need to have `Playwright/test` with a minimum version of 1.41.2 installed as a dev dependency in the package.json file of your plugin.
-
-You can remove any test files (`[filename].spec.js|ts`) that was generated when you installed Playwright. Let's also add the following line to your `.gitignore` file.
-
-```shell title=".gitignore"
-/playwright/.auth/
-```
+`@grafana/plugin-e2e` extends Playwright APIs, so you need to have `@playwright/test` with a minimum version of 1.41.2 installed as a dev dependency in the package.json file of your plugin. Please refer to the [Playwright documentation](https://playwright.dev/docs/intro#installing-playwright) for instruction on how to install. Make sure you can run the example tests that were generated when you installed Playwright before you proceed to the next step in this guide.
 
-## Set up `plugin-e2e`
+## Set up @grafana/plugin-e2e
```

Comment:
```suggestion
## Set up `@grafana/plugin-e2e`
```

Reply:
See comment in GitHub conversations regarding this. Grafana style, following Google, prefers `code formatting` in headers so long as a code term is not used alone.